docs/guides/getting-started/setup/integrate.mdx

```diff
@@ -0,0 +1,184 @@
+  }}
+  beforeDevCommandExplination={{
+    __html:
+      'For the project example in this guide, this is <code>npm run dev</code> (be sure to adapt to use your package manager of choice).',
+  }}
+  beforeBuildCommandExplination={{
+    __html:
+      'For the project example in this guide, this is <code>npm run build</code> (be sure to adapt to use your package manager of choice).',
+  }}
+/>
+
+And that's it, you have now added Tauri to your existing project and you should see a `src-tauri` directory that looks something like this:
+
+```diff {9-14}
+│── package.json
+│── public
+│   ╰── index.html
+│── src
+│   │── App.css
+│   │── App.jsx
+│   │── index.css
+│   ╰── index.js
+╰── src-tauri
+    │── Cargo.toml
+    │── build.rs
+    │── icons
+    │── src
+    ╰── tauri.conf.json
+```
+
+## Invoke Commands
+
+<Commands />
+
+There are two different ways you can invoke commands from your frontend project:
+
+1. Using the [`@tauri-apps/api`] JavaScript library (preferred)
+2. Using [`withGlobalTauri`] to use a pre-bundled version of the Tauri API
+
+We'll go through both below.
+
+### Using JavaScript Library
+
+To call our newly created command we will use the [`@tauri-apps/api`] JavaScript library. It provides access to core functionality such as windows, the filesystem, and more through convenient JavaScript abstractions. You can install it using your favorite JavaScript package manager:
+
+<InstallTauriApi />
+
+You can now import the `invoke` function from the library and use it to call our command:
+
+```jsx title=src/App.jsx {4,7-12}
+import logo from './logo.svg';
+import './App.css';
+
+import { invoke } from '@tauri-apps/api'
+
+function App() {
+  // now we can call our Command!
+  // Right-click the application background and open the developer tools.
+  // You will see "Hello, World!" printed in the console!
+  invoke('greet', { name: 'World' })
+  // `invoke` returns a Promise
+  .then((response) => console.log(response))
+
+  return (
+    // -- snip --
+  )
+}
+```
+
+### Using `withGlobalTauri`
+
+To interact with Tauri from your frontend without using the `@tauri-apps/api` JavaScript package you will need to enable [`withGlobalTauri`] in your `tauri.conf.json` file:
+
+```json title=tauri.conf.json
+{
+  "build": {
+    "beforeBuildCommand": "npm run build",
+    "beforeDevCommand": "npm run dev",
+    "devPath": "http://localhost:3000",
+    "distDir": "../build",
+    // highlight-next-line
+    "withGlobalTauri": true
+  },
+```
+
+This will inject a pre-bundled version of the API functions into your frontend.
+
+You can now modify the `App.jsx` file to call your command:
+
+```jsx title=src/App.js {4,7-12}
+import logo from './logo.svg';
+import './App.css';
+
+// access the pre-bundled global API functions
+const invoke = window.__TAURI__.invoke
```

Comment:
```suggestion
const { invoke } = window.__TAURI__
```

Reply:
This is how it is in the HTML guide, should I adjust it over there too?

Reply:
probably yeah, since this is how I have it in CTA